builder/processor/src/main/java/io/helidon/builder/processor/BlueprintProcessor.java

```diff
@@ -253,331 +259,223 @@ private void generatePrototypeWithBuilder(TypeElement builderInterface,
-            /*
-              static Builder builder()
-             */
-                pw.print(SOURCE_SPACING);
-                pw.println("/**");
-                pw.print(SOURCE_SPACING);
-                pw.println(" * Create a new fluent API builder to customize configuration.");
-                pw.print(SOURCE_SPACING);
-                pw.println(" *");
-                pw.print(SOURCE_SPACING);
-                pw.println(" * @return a new builder");
-                pw.print(SOURCE_SPACING);
-                pw.println(" */");
-                pw.print(SOURCE_SPACING);
-                pw.print("static");
-                if (!blueprintDef.typeArguments().isEmpty()) {
-                    pw.print(" ");
-                    pw.print(blueprintDef.typeArguments());
-                }
-                pw.print(" Builder");
-                pw.print(blueprintDef.typeArguments());
-                pw.println(" builder() {");
-                pw.print(SOURCE_SPACING);
-                pw.print(SOURCE_SPACING);
-                if (blueprintDef.typeArguments().isEmpty()) {
-                    pw.print("return new ");
-                    pw.print(ifaceName);
-                    pw.println(".Builder();");
-                } else {
-                    pw.print("return new ");
-                    pw.print(ifaceName);
-                    pw.println(".Builder<>();");
-                }
-                pw.print(SOURCE_SPACING);
-                pw.println("}");
-                pw.println();
-
-            /*
-              static Builder builder(T type)
-             */
-                pw.print(SOURCE_SPACING);
-                pw.println("/**");
-                pw.print(SOURCE_SPACING);
-                pw.println(" * Create a new fluent API builder from an existing instance.");
-                pw.print(SOURCE_SPACING);
-                pw.println(" *");
-                pw.print(SOURCE_SPACING);
-                pw.println(" * @param instance an existing instance used as a base for the builder");
-                pw.print(SOURCE_SPACING);
-                pw.println(" * @return a builder based on an instance");
-                pw.print(SOURCE_SPACING);
-                pw.println(" */");
-                pw.print(SOURCE_SPACING);
-                pw.print("static");
-                if (!blueprintDef.typeArguments().isEmpty()) {
-                    pw.print(" ");
-                    pw.print(blueprintDef.typeArguments());
+        // prototype interface (with inner class Builder)
+        ClassModel.Builder classModel = ClassModel.builder()
+                .type(prototype)
+                .classType(ClassType.INTERFACE)
+                .licenseHeader(CopyrightHandler.copyright(GENERATOR,
+                                                          typeInfo.typeName(),
+                                                          prototype));
+
+        String javadocString = blueprintDef.javadoc();
```

Comment:
shouldn't this be an `Optional`, how can this ever be `null`?

Reply:
`javadoc` method can return null in cases where described class/interface does not have any javadoc present. And I believe this `javadoc` method does not need to return `Optional` since it is package private and `Optional` is required only for public methods.